### PR TITLE
complete 6-get-article-comments

### PR DESCRIPTION
**Pull request by ramemes**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -105,4 +105,38 @@ describe("GET/api/articles", () => {
...
+        return request(app).get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            body.comments.forEach((comment) => {
+                expect(typeof comment.comment_id).toBe("number")
+                expect(typeof comment.votes).toBe("number")
+                expect(typeof comment.created_at).toBe("string")
+                expect(typeof comment.author).toBe("string")
+                expect(typeof comment.body).toBe("string")
+                expect(typeof comment.article_id).toBe("number")
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:121]: Please test for the exact article_id here

File: __tests__/app.test.js [hunk 2]
```diff
@@ -105,4 +105,38 @@ describe("GET/api/articles", () => {
...
+            }) 
+            expect(body.comments.length).toBe(11)   
+        })
+    })
+    test("returns correct error if article_id doesnt exist", () => {
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:126]: Tests descriptions need to be a bit more specific. Instead of saying 'correct error' .

File: __tests__/app.test.js [hunk 3]
```diff
@@ -105,4 +105,38 @@ describe("GET/api/articles", () => {
...
+    })
+
+    test("returns correct error if incorrect format given", () => {
+        return request(app).get('/api/articles/dasads/comments')
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("invalid article id format")   
+        })
+    })
+
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:141]: Please add another test where there are no comments for the article and it responds with an empty array and status 200.

File: controllers/articles.controllers.js [hunk 4]
```diff
@@ -20,7 +20,17 @@ exports.getArticles = async (req, res, next) => {
         res.status(200).send({articles: articles})
     }
     catch(err) {
-        console.log(err)
         next(err)
     }
 }
+
+exports.getArticleComments = async (req, res, next) => {
+    try {
```
Comment (nc-bela-bertalan) [inline controllers/articles.controllers.js:28]: Here you need to check if the article ID exists, otherwise how would you tell if the article doesn't exist or just no comments for the article?

File: models/articles.models.js [hunk 5]
```diff
@@ -27,6 +27,17 @@ exports.fetchArticles = async () => {
...
+    return queryResponse.rows
+}
+
+exports.fetchArticleComments = async (article_id) => {
+    const queryResponse = await db.query(`
+    SELECT * FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC
+    `, [article_id])
+    if (queryResponse.rows.length === 0) {
```
Comment (nc-bela-bertalan) [inline models/articles.models.js:39]: Empty rows of comments. Does it mean no article with that it? Or does it mean no comments for the existing article yet?

**Review by nc-bela-bertalan — COMMENTED** [review]
Hey Raheem, please see my comments.
